script/brew-publish

```diff
@@ -46,7 +46,8 @@ rm -f "${formula}.bak"
 git commit -m "${brew_name} ${version#v}" -- "$formula"
 # hackish way of getting the git remote name for user's fork
 fork_remote="$(hub fork 2>&1 | grep -oE 'remote:? \S+' | tail -1 | awk '{print $2}')"
-git push -u "$fork_remote" "$branch"
+git push -u "${fork_remote}" "${branch}:${branch}"
```

Comment:
Since we're currently on the branch that we want to publish, how about using `HEAD` as refspec? Git documentation says that it will always publish the same-named branch on the remote.